public/locales/gr/common.json

```diff
@@ -0,0 +1,25 @@
+
+{
+"logo": {
+"alt":"Το λογότυποτου Instant Visio, αντιπροσωπεύει μια μπλε οθόνη κινητού τηλεφώνου με πρόσωπο σε μορφή πορτοκαλί κύκλου, μιλάει οπτικά στον ιδιοκτήτη του τηλεφώνου, του οποίου το πρόσωπο εμφανίζεται με τη μορφή κύκλου με πορτοκαλί άκρες.”
+},
+"homeBaseline": "Συμμετάσχετε με ένα αγαπημένο σας πρόσωπο με βίντεο, με ένα κλικ, <1> δωρεάν. </ 1>",
+"headerBaseline": "Συμμετάσχετε με ένα αγαπημένο σας πρόσωπο με βίντεο, με ένα κλικ, <1> δωρεάν. </ 1>",
+"footer": {
+"contact-us": "Επικοινωνήστε μαζί μας",
+"legal-mentions":"Νομικές ειδοποιήσεις",
+"personal-data": "Προσωπικά δεδομένα",
+"blog":
+ "Blog",
+"credits": "Crédits",
+
+"contact":"Επικοινωνία"
+},
+"url": {
+"legal-mentions":"Νομικές ειδοποιήσεις",
+"personal-data": "Προσωπικά δεδομένα",
+"blog": "blog",
+"blog-article":"blog /: άρθρο",
```

Comment:
As changes were since made to the French version of this file (available here: https://github.com/Instant-Visio/InstantVisio-WebApp/blob/81ab76abe8653c9f972035ff87b4bccf7f3d1e6b/public/locales/de/common.json), could you, in the url object:

remove the "blog-article" key
add "video-call": "visio"